.prettierrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "plugins": ["prettier-plugin-svelte"],
+  // No reason other than the JSON file was already formatted with 4 spaces
+  // when we made this prettier file
+  "overrides": [
+    {
+      "files": "*.json",
+      "options": {
+        "tabWidth": 4
+      }
+    }
+  ]
+}
```

Comment:
Using the .js format since we can add comments. Only practical change is that the json file is formatted automatically now.